tiny_dnn/activations/activation_function.h

```diff
@@ -165,7 +196,16 @@ class tan_h : public function {
   x));
   }*/
 
-  float_t df(float_t y) const override { return float_t(1) - sqr(y); }
+  inline float_t df(float_t y) const override { return float_t(1) - sqr(y); }
+  void itedf(vec_t &cur,
+             const vec_t &prev,
+             const vec_t &out,
+             size_t cnt) const {
+    for (size_t i = 0; i < cnt; i++) {
+      cur[i] = prev[i] * df(out[i]);
```

Comment:
@Randl For the same rule it should do the same thing here Could you check it?
Besides I think that some original codes were like this http://stackoverflow.com/questions/32682426/clang-format-align-function-parameter-names